cookbooks/llama/typescript/ask-llama.ts

```diff
@@ -1,5 +1,6 @@
 import { AIConfigRuntime } from "aiconfig";
```

Comment:
Not for this diff, but @rholinshead can you add a video to the cookbook readme showing ask-llama in action? Can be pretty basic, but visual would be useful